addons/show-bbcode/userscript.js

```diff
@@ -1,8 +1,13 @@
-function viewSource(post, msg) {
+function viewSource(post, msg, addon) {
```

Comment:
Could this function be moved inside the main one instead? That would allow it to use `msg` and `addon` without needing to pass it in.

Reply:
Done